packages/url_launcher/lib/url_launcher.dart

```diff
@@ -30,6 +30,9 @@ const MethodChannel _channel = MethodChannel('plugins.flutter.io/url_launcher');
 /// Note that if any of the above are set to true but the URL is not a web URL,
 /// this will throw a [PlatformException].
 ///
+/// [enableJavaScript] is an Android only setting. If true, and [forceWebView]
```

Comment:
I would put this description together with `forceWebView` description in the same paragraph.